src/components/common/layout/base.njk

```diff
@@ -92,7 +87,7 @@
                     text: 'general.footer.accessibilityStatement.linkText' | translate
                 },
                 {
-                    href: "https://www.gov.uk/help/cookies",
+                    href: "/cookies",
```

Comment:
cookies page is on the frontend so I think it's:

href: authFrontEndUrl + "/cookies"

Reply:
Yes, you're right of course. Thanks